backend/server/schema.py

```diff
@@ -1,17 +1,28 @@
-from typing import List, cast
+from typing import List, cast, Optional
```

Comment:
why `cast` with all lowercase and the others Capitol? is this from another external package or yours? if yours, I would keep it consistent if possible. or if apply.

Reply:
`typing` is a module in the Python standard library: it's a function rather than a data type, which is why it's lowercase